spec/models/bookings/registration_contact_mapper_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Bookings::RegistrationContactMapper do
+  describe ".new" do
+    let(:registration) { build(:registration_session) }
+    let(:contact) { Bookings::Gitis::Contact.new }
+    subject { described_class.new(registration, contact) }
+
+    it { is_expected.to have_attributes(registration_session: registration) }
+    it { is_expected.to have_attributes(gitis_contact: contact) }
+  end
+
+  describe "#registration_to_contact" do
+    let(:registration) { build(:registration_session) }
+    let(:contact) { Bookings::Gitis::Contact.new }
+    let(:mapper) { described_class.new(registration, contact) }
+    subject { mapper.registration_to_contact }
+
+    it { is_expected.to have_attributes(firstname: registration.personal_information.first_name) }
+    it { is_expected.to have_attributes(lastname: registration.personal_information.last_name) }
+    it { is_expected.to have_attributes(email: registration.personal_information.email) }
+    it "needs to compare date of birth when that is merged in"
```

Comment:
Ha, think we're both waiting on each other. In which case I'll cherry pick it into my branch - there's some hacking around to do since I've reworked the various screens. Thanks